guide/05-working-with-the-spatially-enabled-dataframe/part1_introduction_to_sedf.ipynb

```diff
@@ -0,0 +1,1555 @@
+{
```

Comment:
<p>Let's <strong>now </strong>perform some basic Pandas operations on a SeDF. <strong>One of the benefits of the accessor pattern in SeDF is, the SeDF object is still of type 'DataFrame'. Thus, you can continue to perform regular Pandas DataFrame operations.</strong></p><p> </p><p>We will:</p><p>....</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>